src/file.spec.ts

```diff
@@ -56,6 +59,25 @@ describe('toFile()', () => {
         expect(fwp.lastModified).toEqual(file.lastModified);
     });
 
+    it('clones the File as a blob', () => {
+        const opts: FilePropertyBag = {
+            type: 'plain/text',
+            lastModified: 1234567
+        };
+        const data = JSON.stringify({ping: true});
+        const file = new File([data], 'test.txt', opts);
+        global.OriginalFile = File;
+        global.File = MockFile;
+        const clonedFile = clone(file);
+        global.File = (global as any).OriginalFile;      
+
+        expect(clonedFile === file).toBe(false);
```

Comment:
I'd add a test to check if the cloned file is an instance of `Blob` (which should be true).

Reply:
Clone function returns Blob as File, so it is an actual File when it's returned. (I had that test in there at first, but forgot to read the code)

Reply:
`as File` is just a type hint for TypeScript, it does not have any effect on the `instanceof` evaluation. If it does, it means that the catch block is not evaluated.

Reply:
Okay, I'll see if I can add that expect.